src/ensembl/src/shared/components/overlay/Overlay.scss

```diff
@@ -7,5 +7,5 @@
   left: 0;
   right: 0;
   background-color: rgba(255, 255, 255, 0.7);
-  z-index: 1;
+  z-index: 11; /* Need to go above the top bar */
```

Comment:
Above what, sorry? I tried to dial the z-index back to 1, and didn't see any element going above it.

![image](https://user-images.githubusercontent.com/6834224/127156662-49b6ac7b-61f0-4d00-a3b0-ca7c58b29fb4.png)

Reply:
The topbar css defined here has z-index 10:
https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/components/layout/StandardAppLayout.scss#L21

Reply:
I see 👍